CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 - `ext:install` now supports `--force` and `--non-interactive` flags.
 - Fixes a crash when customers deploy an empty Cloud Functions project (#3705)
+- Adds command for listing all functions in the Firebase project `functions:list`.
```

Comment:
```suggestion
- Adds a command (`functions:list`) for listing all functions in the Firebase project.
```

Also, is it "all Google Cloud Functions" or "all Cloud Functions for Firebase" in the project?

Reply:
We'd expect this to usually be all Cloud Functions for Firebase since the user is running this command in their Firebase project. However, under the hood this command calls the GCP REST API for obtaining all Google Cloud Functions. So if a user creates a Google Cloud Function in the same project as they deploy Firebase Functions, then this command will display that function.

Reply:
FWIW - Firebase Console also displays all GCFs on the user project; it doesn't try to filter anything.